test/unit/src/signals/SignalsTest.cpp

```diff
@@ -102,8 +102,9 @@ TEST_CASE( "signals/Signals" )
 	{
 		class MemberClass {
 		public:
+			MemberClass() { mAccum = 0; }
 			void handler()	{ mAccum++;	}
-			int mAccum = 0;
```

Comment:
I believe it's called ['Non-static data member initializers'](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2008/n2756.htm) and according to [this](https://msdn.microsoft.com/en-us/library/hh567368.aspx), they are supported in VS2013. But maybe it is because `MemberClass` is declared locally and VS2013 just has a bug related to that?

Anywho, change doesn't effect the test much.